web/packages/core/package.json

```diff
@@ -10,7 +10,7 @@
     ],
     "scripts": {
         "build": "npm run build:cargo && npm run build:wasm-bindgen && npm run build:wasm-opt && npm run build:ts",
-        "build:cargo": "cross-env-shell \"cargo build --release --target wasm32-unknown-unknown --features \\\"$CARGO_FEATURES\\\"\"",
+        "build:cargo": "cross-env-shell RUSTFLAGS=\"-C target-feature=+bulk-memory\" \"cargo build --release --target wasm32-unknown-unknown --features \\\"$CARGO_FEATURES\\\"\"",
```

Comment:
I think it would be better to add this flag to the top-level `Cargo.toml`:
```toml
[target.wasm32-unknown-unknown]
rustflags = ["-C target-feature=+bulk-memory"]
```
(Haven't had an oppotunity to test it though)

Reply:
The top-level `Cargo.toml` didn't accept it:
```
ruffle-core: error: failed to parse manifest at `.../ruffle/Cargo.toml`
ruffle-core: Caused by:
ruffle-core:   this virtual manifest specifies a [target] section, which is not allowed
```

Probably because this file describes a `[workspace]`?

Reply:
And it I put it into the separate `Cargo.toml` files of the relevant packages:
```
ruffle-core: warning: .../ruffle/web/Cargo.toml: unused manifest key: target.wasm32-unknown-unknown.rustflags
ruffle-core: warning: .../ruffle/core/Cargo.toml: unused manifest key: target.wasm32-unknown-unknown.rustflags
ruffle-core: warning: .../ruffle/render/webgl/Cargo.toml: unused manifest key: target.wasm32-unknown-unknown.rustflags
ruffle-core: warning: .../ruffle/render/canvas/Cargo.toml: unused manifest key: target.wasm32-unknown-unknown.rustflags
```

Reply:
Not to mention that the `release_nightly` workflow already sets `$RUSTFLAGS` on windows, so, either way, something is bound to conflict with something else... :/

Reply:
According to  https://users.rust-lang.org/t/rustflags-via-cargo/38348/2: 
> These options need to go into a .cargo/config file (or $CARGO_HOME/config, see the first part on Cargo book - Configuration 15).

https://doc.rust-lang.org/cargo/reference/config.html#configuration

Reply:
For the record, when I was testing these features, I also used RUSTFLAGS.

(I really wish the toolchain (not just rustc, but also wasm-pack etc) had better support for wasm extensions, currently we break right into LLVM features system...)

Reply:
wasm-pack...? I don't see any relevant flag of that tool. Did you perhaps mean wasm-opt?